### PR TITLE
Updated Compose to 2024.09.00

### PR DESCRIPTION
**Pull request by kamilwygralak-au**
- updated Compose
- fixed build issues in modules with sample code, because it somehow affected the Jitpack build

### CODE REVIEW

File: amity-sample-code/build.gradle [hunk 1]
```diff
@@ -37,6 +37,9 @@ android {
 
 dependencies {
     api project(path: ':amity-uikit')
+    // Added :chat module manually as it is unused in base amity-uikit module
+    implementation project(path: ':chat')
```
Comment (lonskiTomasz) [inline amity-sample-code/build.gradle:41]: [question]
Do we use it? Or, without it, are some build errors?

File: sample/build.gradle [hunk 2]
```diff
@@ -2,8 +2,8 @@ apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
```
Comment (lonskiTomasz) [inline sample/build.gradle:2]: Maybe we could just remove the whole sample? 🤔

**Review by lonskiTomasz — APPROVED** [review]

**Review by marcin-adamczewski — APPROVED** [review]

File: amity-sample-code/build.gradle (hunk 1 above)
Comment (kamilwygralak-au) [reply amity-sample-code/build.gradle:41]: We don't use it, but there were build errors and sometimes it affects the build on Jitpack. Some modules are compiled in parallel and if one of them fails, then the rest is cancelled.

File: sample/build.gradle (hunk 2 above)
Comment (kamilwygralak-au) [reply sample/build.gradle:2]: not a big deal to maintain it and it is sometimes useful to build it locally and check out the sample

**Pull request merged**